### DISCUSSION TITLE
Reminder Untuk Punch Out Pada Absensi

### DISCUSSION THREAD

**Discussion by melodiazz** [kirimwa-id/forum#45] [Pamerkan]
![image](https://user-images.githubusercontent.com/22655268/135700559-5ebd5f0d-4556-469e-ad07-65a03ff517a7.png)

Reminder untuk karyawan yang belum melakukan punch out pada sistem absensi online. Thanks pak @rioastamal!

**Comment by rioastamal** [comment]
@melodiazz Cool 😎 . Terima kasih sudah menginfokan penggunaan API KirimWA.id ditunggu update penggunaan-pengguaan yang lain.
